app/cli.php

```diff
@@ -17,10 +19,13 @@
 use Utopia\Database\Document;
 use Utopia\Logger\Log;
 use Utopia\Pools\Group;
+use Utopia\Queue\Connection;
 use Utopia\Registry\Registry;
 
 Authorization::disable();
 
+global $register;
```

Comment:
Why are we adding a global here? Shouldn't getResource('register') be all we need?

Reply:
It does; it is mainly for the editor.
Removed it